mozci/task.py

```diff
@@ -206,19 +216,27 @@ def __post_init__(self):
             assert self._results is None, f"{self.label} should have no results"
             self._results = []
 
+        if self._results is None:
+            return
+
         # XXX: A while after bug 1613937 and bug 1613939 have been fixed, we can
         # remove the filtering and slash replacing.
 
-        def update_group(result):
+        # Apply WPT workaround, needed at least until bug 1632546 is fixed.
+        if self.is_wpt:
+            for result in self._results:
+                result.group = wpt_workaround(result.group)
```

Comment:
It looks like this means we only apply the workaround if we pass in the results from the constructor (i.e data was loaded from AD). Could you refactor this so it also gets applied when loaded via the errorsummary?

I realize that we already had this problem to an extent with the replacing of slashes. So feel free to file a follow-up issue if you prefer.

Reply:
We are actually calling the `__post_init__` function manually in `_load_errorsummary` (it's a bit of a hack, I know). I can move the clean-up code in a separate function if you want and call it both from `__post_init__` and `_load_errorsummary` (removing the `__post_init__` call from `load_errorsummary`).

Reply:
I'll just merge as-is, we can clean up later if we want.